### PR TITLE
Martial and Ovid and Juvenal

### PR DESCRIPTION
**Pull request by PonteIneptique**
@balmas Even if I already started, I'd like to have your confirmation for this PR. The metadata file should be changed, but it has not that much importance right now...

### CODE REVIEW

File: CTS_XML_TEI/perseus/latinLit/phi0959/.cts.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ti:textgroup
+  xmlns:ti="http://chs.harvard.edu/xmlns/cts"
+  urn="urn:cts:latinLit:phi0959"
+  tiid="annotsrc">
```
Comment (balmas) [inline CTS_XML_TEI/perseus/latinLit/phi0959/.cts.xml]: hmm. are we going to have this specified in these files? Isn't the idea that they can be used in any aggregate inventory?

File: CTS_XML_TEI/perseus/latinLit/phi0959/phi001/.cts.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <ti:title xml:lang="eng">Amores</ti:title>
+  <ti:edition
+    xmlns:ti="http://chs.harvard.edu/xmlns/cts"
+    xml:lang="lat"
+    urn="urn:cts:latinLit:phi0959.phi001.perseus-lat1"
+    workUrn="urn:cts:latinLit:phi0959.phi001">
+    <ti:label xml:lang="eng">Amores</ti:label>
+    <ti:online docname="/db/repository/latinLit/phi0959/phi001/perseus-lat1.xml">
+      <ti:namespaceMapping abbreviation="tei" nsURI="http://www.tei-c.org/ns/1.0"/>
+      <ti:citationMapping>
```
Comment (balmas) [inline CTS_XML_TEI/perseus/latinLit/phi0959/phi001/.cts.xml]: we decided citationMapping would come from the refsDecl right?

File: CTS_XML_TEI/perseus/latinLit/phi0959/phi003/phi0959.phi003.perseus-lat1.xml [hunk 3]
```diff
@@ -45,149 +47,132 @@
...
-				<refState unit="line"/>
-			</refsDecl>
-			<refsDecl n="text=Ars">
-				<refState unit="text"/>
-				<refState unit="book"/>
-				<refState unit="line"/>
-			</refsDecl>
-			<refsDecl n="text=Rem.">
-				<refState unit="text"/>
+			<refsDecl>
```
Comment (balmas) [inline CTS_XML_TEI/perseus/latinLit/phi0959/phi003/phi0959.phi003.perseus-lat1.xml]: we need a refsDecl type = cts??

File: CTS_XML_TEI/perseus/latinLit/phi0959/phi003/phi0959.phi003.perseus-lat1.xml [hunk 4]
```diff
@@ -45,149 +47,132 @@
...
-				<l rend="indent">Fissaque adoptivas accipit arbor opes.</l>
-				<l>Culta placent. auro sublimia tecta linuntur,</l>
-				<l rend="indent">Nigra sub imposito marmore terra latet:</l>
-				<l>Vellera saepe eadem Tyrio medicantur aëno:</l>
-				<l n="10" rend="indent">Sectile deliciis India praebet ebur.</l>
-				<l>Forsitan antiquae Tatio sub rege Sabinae</l>
-				<l rend="indent">Maluerint, quam se, rura paterna coli:</l>
-				<l>Cum matrona, premens altum rubicunda sedile,</l>
-				<l rend="indent">Assiduum duro pollice nebat opus,</l>
+				<l n="6">Fissaque adoptivas accipit arbor opes.</l>
```
Comment (balmas) [inline CTS_XML_TEI/perseus/latinLit/phi0959/phi003/phi0959.phi003.perseus-lat1.xml]: we lost the rend=indent....

**Comment by balmas** [comment]
@PonteIneptique if I understand correctly and you aren't intending this to be the final format of the metadata or full epidoc compliance, basically just getting these texts CTS-ized, with entities and betacode fixed. Is that right?

If so, then the only thing I think that needs to be fixed before merging is the lost of the indent attribute on the lines.

**Comment by lcerrato** [comment]
Please preserve as much header info as possible.
Couple of minor notes.
why is publisher being changed to Leipzig University (note there is a typo) from Leipzig? Book reads Leipzig, I believe.
why was the initial markup info changed/removed? It read that Greg (GRC) did the initial markup and that appears to be gone now.
We know that much of the text headers need revision but we plan on doing this in a comprehensive way. If there are specific questions about a text, happy to check these as needed.
Also curious about the alternate titles.

**Comment by lcerrato** [comment]
Addendum: looks to be a Teubner version published in Leipzig.

**Comment by PonteIneptique** [comment]
So, concerning the CTS/RefsDecl, this commit does not concern this part, its purely ctsizing. The metadata files are rests from a former tentative. I'll remove them in a new commit.

For the rest, the rend="indent" is my fault. I didn't feel concern by a typographic attribute like that and should have (but still, what's the point of this attribute is a discussion I am wishing to have, to understand a point)

@lcerrato : Can you point to me which file you are talking about ? All of them ? Might be a huge mistake from my side.

**Comment by nevenjovanovic** [comment]
Thibault, my opinion is that, as generally in verse @rend="indent" marks a specific verse type, keeping this markup is of potential use for limiting analysis to pentameters etc. It is easier to reuse @rend than to select every other verse, because then you have first to exclude hexameter poetry etc.

**Comment by PonteIneptique** [comment]
Hey @nevenjovanovic, thanks for stopping by. Can I ask you why then we don't use directly a metric information ? Or maybe in a mail :')

**Comment by PonteIneptique** [comment]
Ok @lcerrato, my bad for the metadata. I am just feeding them back as I see. The only metadata I removed without doubts and intentionnaly is the "lang=greek" you can see in Ovid mostly, because it was due to the original not splitted file.

**Comment by PonteIneptique** [comment]
Also thanks for the meaning of GRC, which I am putting back instead of "unknown".

**Comment by PonteIneptique** [comment]
So, @balmas, @lcerrato and probably @simonastoyanova / @srdee (That's a lot, sorry for the spam) : the epigrammata file gave me some Epidoc troubles, could you have a look ?

**Comment by PonteIneptique** [comment]
Ok, there is epidoc error all over the place, maybe we could plan a call in the afternoon ( @simonastoyanova , @lcerrato , @balmas ? Who is the best to ask and the one available ?)

**Comment by PonteIneptique** [comment]
@balmas Concerning the __cts__.xml files, you have the real form here : https://github.com/PonteIneptique/cts-metadata

**Comment by balmas** [comment]
I think epidoc conversion should be handled separately because we are minting new version level URNs for the epidoc versions.

**Comment by PonteIneptique** [comment]
Note for anyone interested : putting back indent in Oxygen

**Search (ctr+f)**
**Find** : `n="([0-9]+)"`
**Replace with**: `n="$1" render="indent"`
**xpath** : `//l[@n and @n mod 2 = 0]`
**Regular Expression** : Checked

**Comment by PonteIneptique** [comment]
Ready for merge. Normally, data has been put back as far as I can see. :)

**Comment by PonteIneptique** [comment]
Because this commit list is huge, I am creating a new PR

**Pull request closed without merging**